src/modules/languages/ui/languages-dialog.tsx

```diff
@@ -0,0 +1,50 @@
+import { useTranslation } from "react-i18next";
+import { Button } from "antd";
+import { Form } from "@/components/form";
+import { FormTextField } from "@/components/form-text-field";
+import { createDialogHook } from "@/helpers/dialog/create-dialog";
+import { BaseDialog } from "@/components/base-dialog/";
+import { FormSubmitButton } from "@/components/form-submit-button";
+import { getLanguageFormSchema } from "../shemas/language";
+
+interface FormData {
+    language: string;
+    native_name: string;
+    iso2: string;
+}
+interface LanguageDialogProps {
+    title: string;
+    onConfirm: (formData: FormData) => void;
+    onClose: () => void;
+    initialValues: { language: string; native_name: string; iso2: string };
```

Comment:
this type is the same as `FormData` interface above

```suggestion
    initialValues: FormData;
```